docs/targets/elasticsearch.md

```diff
@@ -1,75 +1,58 @@
-# Event target for Elasticsearch
+# Event Target for Elasticsearch
 
-This event target receives [CloudEvents][ce] over HTTP and index their payload intoElasticsearch.
+This event Target receives [CloudEvents][ce] over HTTP and indexs their payload into [Elasticsearch][es]
 
 ## Prerequisites
 
-A Elasticsearch cluster and a set of credentials for indexing documents.
+1. If there is not a pre-existing Elasticsearch cluster available, get started quickly by using either [Elastic Cloud on Kubernetes][eck] or [Elastic Cloud][elasticcloud]
```

Comment:
"There is no"

Reply:
Missing dot.

Reply:
@cab105 do we need to replace the 2nd person in this imperative? If we do, the whole page needs a rewrite.

See also my comment on #34.

Reply:
More than likely not in this case, however as per your comment https://github.com/triggermesh/docs/pull/29/files#r466502571 this should be consistent in the usage.